tests/output_manager.py

```diff
@@ -58,7 +58,7 @@ def testWrite(self):
     with open(src, 'w') as file_handle:
       file_handle.write(contents)
 
-    self.assertTrue(writer.write(src))
```

Comment:
Is this same as https://github.com/google/turbinia/pull/178?

Reply:
Heh, it is now because you merged the other PR, and the base was set to this branch :).  I'll talk to you more about it out of band.